modules/learn/pages/clusters-and-availability/automatic-failover.adoc

```diff
@@ -224,15 +224,9 @@ xref:cli:cbcli/couchbase-cli-setting-autofailover.adoc[setting-autofailover] wit
 [#auto-failover-during-rebalance]
 == Auto-Failover During Rebalance
 
-Couchbase Server provides a setting to determine whether, once enabled, auto-failover should specifically be triggered during xref:learn:clusters-and-availability/rebalance.adoc[Rebalance], in the event of a node becoming unresponsive.
-
-If auto-failover _has_ been set to be triggered, following the configured timeout period, the rebalance is stopped; then, auto-failover is duly triggered.
 Following auto-failover, rebalance is _not_ automatically re-attempted.
 At this point, the cluster is likely to be in an unbalanced state: therefore, rebalance should be performed manually; and the unresponsive node fixed and restored to the cluster, as appropriate.
 
-If auto-failover has _not_ been set to be triggered, unless there is manual intervention, no failover occurs.
-This may cause the rebalance to hang for an indeterminate period; before failing, with error messages.
-
 For information on setting auto-failover in the context of rebalance, see the information on xref:manage:manage-settings/general-settings.adoc[General] settings.
```

Comment:
This line -- the current line 230 -- "For information on setting auto-failover in the context of rebalance, ... " should also be removed since users can no longer set/unset auto-failover in the context of rebalance.

Reply:
Done 👍🏾